src/app/paymentSlice.js

```diff
@@ -30,31 +28,12 @@ export const slice = createSlice({
     },
   },
   reducers: {
-    paymentMethods: (state, action) => {
+    paymentSession: (state, action) => {
       const [res, status] = action.payload;
       if (status >= 300) {
         state.error = res;
       } else {
-        res.paymentMethods = res.paymentMethods.filter((it) =>
-          ["eps", "scheme", "dotpay", "giropay", "ideal", "directEbanking", "bcmc", "paysafecard"].includes(it.type)
-        );
-        state.paymentMethodsRes = res;
-      }
-    },
-    payments: (state, action) => {
-      const [res, status] = action.payload;
-      if (status >= 300) {
-        state.error = res;
-      } else {
-        state.paymentRes = res;
-      }
-    },
-    paymentDetails: (state, action) => {
-      const [res, status] = action.payload;
-      if (status >= 300) {
-        state.error = res;
-      } else {
-        state.paymentDetailsRes = res;
+        state.sessionAndOrderRef = res;
```

Comment:
Maybe split up this field? `[state.session, state.orderRef] = res;`

Reply:
We use them together in the payload later, they don't make sense apart form each other. good idea otherwise